modules/ztp-pre-hub-install-factory-pipeline-checks.adoc

```diff
@@ -0,0 +1,103 @@
+cloud-controller-manager                    4.10.9   True        False         False      3h
+cloud-credential                            4.10.9   True        False         False      179m
+cluster-autoscaler                          4.10.9   True        False         False      178m
+config-operator                             4.10.9   True        False         False      3h
+console                                     4.10.9   True        False         False      168m
+csi-snapshot-controller                     4.10.9   True        False         False      178m
+dns                                         4.10.9   True        False         False      178m
+etcd                                        4.10.9   True        False         False      177m
+image-registry                              4.10.9   True        False         False      172m
+ingress                                     4.10.9   True        False         False      173m
+insights                                    4.10.9   True        False         False      172m
+kube-apiserver                              4.10.9   True        False         False      175m
+kube-controller-manager                     4.10.9   True        False         False      176m
+kube-scheduler                              4.10.9   True        False         False      175m
+kube-storage-version-migrator               4.10.9   True        False         False      179m
+machine-api                                 4.10.9   True        False         False      175m
+machine-approver                            4.10.9   True        False         False      179m
+machine-config                              4.10.9   True        False         False      102m
+marketplace                                 4.10.9   True        False         False      178m
+monitoring                                  4.10.9   True        False         False      93m
+network                                     4.10.9   True        False         False      3h
+node-tuning                                 4.10.9   True        False         False      178m
+openshift-apiserver                         4.10.9   True        False         False      173m
+openshift-controller-manager                4.10.9   True        False         False      174m
+openshift-samples                           4.10.9   True        False         False      172m
+operator-lifecycle-manager                  4.10.9   True        False         False      179m
+operator-lifecycle-manager-catalog          4.10.9   True        False         False      178m
+operator-lifecycle-manager-packageserver    4.10.9   True        False         False      173m
+service-ca                                  4.10.9   True        False         False      179m
+storage                                     4.10.9   True        Flase         False      179m
+----
+
+. Verify that enough persistent volumes exist and are available:
```

Comment:
It seems like this step might be missing a command?

Reply:
good catch - thanks